src/main.ts

```diff
@@ -10,41 +10,45 @@ import * as xpath from 'xpath'
 import {JSONPath} from 'jsonpath-plus'
```

Comment:
Remove `/* eslint-disable no-console */` at top

Reply:
This isn't written for exclusive use in Actions so it does provide some console output.